bookshelf/src/app.php

```diff
@@ -39,8 +39,11 @@
 // register the url generator
 $app->register(new UrlGeneratorServiceProvider);
 
+$config = getenv('BOOKSHELF_CONFIG') ?: 'settings.yml';
```

Comment:
Isn't it tricky to do that? I mean, the full path will be `/app/config/settings.yml` and of course differs in the local environment.